Caboodle/DataTransfer/DataTransfer.uwp.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.DataTransfer;
+using Windows.Foundation;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class DataTransfer
+    {
+        public static Task RequestAsync(ShareTextRequest request)
+        {
+            var dataTransferManager = DataTransferManager.GetForCurrentView();
+
+            dataTransferManager.DataRequested += new TypedEventHandler<DataTransferManager, DataRequestedEventArgs>(ShareTextHandler);
```

Comment:
What's the lifecycle of the event handler here?

Reply:
ahhh good cactch, need to remove it after it comes back.